spork/data.janet

```diff
@@ -0,0 +1,88 @@
+###
+### data.janet 
+###
+### Compare data structures using `diff`.
+###
+
+(varfn diff [])
+
+(defn- atom-diff
+  [a b]
+  (if (= a b) @[nil nil a] @[a b nil]))
+
+(defn- in? [x ds]
+  (if (index-of x ds)
+    true
+    false))
+
+(defn- safe-in [ds n]
+  (if (in? (type ds) [:array :tuple])
+    (if (<= n (dec (length ds)))
+      (in ds n)
+      nil)
+    (in ds n)))
+
+(defn- arr [a]
+  (apply array a))
```

Comment:
Here we can splice too .

Reply:
Using splice makes this function so trivial it may as well be inlined. Thanks for the improvement!